schemas/com.snowplowanalytics.snowplow.media/ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
```

Comment:
Not sure how to reference the new name unambiguously. :sweat_smile: `com.snowplowanalytics.snowplow.media/ad` is a bit more of a mouthful.

Reply:
The `media_player_ad` was a name of what I originally planned to be a single schema for all ad events but we have diverged from that so just changed it to "ad events".